CHANGELOG.md

```diff
@@ -10,29 +10,44 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Factored out writing of Linux worker start scripts into a library, and added a standalone `WriteLinuxScript.exe` to _just_ write the launch script (for use in custom build pipelines).
 - Added temporary MaxNetCullDistanceSquared to SpatialGDKSettings to prevent excessive net cull distances impacting runtime performance. Set to 0 to disable.
+- Added `OnWorkerFlagsUpdated`, a delegate that can be directly bound to in C++. To bind via blueprints you can use the blueprint callable functions `BindToOnWorkerFlagsUpdated` and `UnbindToOnWorkerFlagsUpdated`. You can use `OnWorkerFlagsUpdated` to register when worker flag updates are received, which allows you to tweak values at deployment runtime.
 
 ### Bug fixes:
+- The `improbable` namespace has been renamed to `SpatialGDK`. This prevents namespace conflicts with the C++ SDK.
 - Disconnected players no longer remain on the server until they time out if the client was shut down manually.
 - Fixed support for relative paths as the engine association in your games .uproject file.
 - RPCs on `NotSpatial` types are no longer queued forever and are now dropped instead.
 - Fixed issue where an Actor's Spatial position was not updated if it had an owner that was not replicated.
-- BeginPlay is only called once with authority per deployment for startup actors
+- BeginPlay is now only called with authority on startup actors once per deployment.
 - Fixed null pointer dereference crash when trying to initiate a Spatial connection without an existing one.
 - URL options are now properly sent through to the server when doing a ClientTravel.
-- Fixed an issue that stopped the correct error message from being shown when the SchemaDatabase is missing.
-- Fixed an issue with `StartEditor.bat` not being generated correctly when building the server worker for local deployments outside of editor.
+- The correct error message is now printed when the SchemaDatabase is missing.
+- `StartEditor.bat` is now generated correctly when you build a server worker outside of editor.
 - Fixed an issue with logging errored blueprints after garbage collection which caused an invalid pointer crash.
 - Removed the ability to configure snapshot save folder. Snapshots should always be saved to `<ProjectRoot>/spatial/snapshots`. This prevents an issue with absolute paths being checked in which can break snapshot generation.
+- Introduced a new module, `SpatialGDKServices`, on which `SpatialGDK` and `SpatilGDKEditorToolbar` now depend. This resolves a previously cyclic dependency.
+- RPCs recieved on entity creation are now queued in case they cannot yet be executed. Previously they were simply dropped.
```

Comment:
```suggestion
- RPCs called before entity creation are now queued in case they cannot yet be executed. Previously they were simply dropped. These RPCs are also included in RPC metrics.
```